contents/docs/product-analytics/funnels.mdx

```diff
@@ -83,12 +85,19 @@ Select the steps to include in your funnel, ideally starting with the first even
     alt="Funnel steps"
 />
 
-By default, funnel steps must be completed sequentially, however you are able to change this behavior by selecting one of three different step orders:
+By default, funnel steps must be completed **sequentially**, however you are able to change this behavior by selecting one of three different step orders:
 
 1. **Sequential** – Step B must happen after Step A, but any number events can happen between A and B.
 2. **Strict order** – Step B must happen directly after Step A without any events in between.
 3. **Any order** – Steps can be completed in any sequence.
 
+<CalloutBox icon="IconInfo" title="Steps must be performed" type="info">
```

Comment:
I think callout box needs to be imported btw

<img width="1392" height="830" alt="CleanShot 2025-10-17 at 10 23 48@2x" src="https://github.com/user-attachments/assets/abad745c-f142-4337-92ba-f61671517e19" />

Reply:
oh 👍 good catch